lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -555,6 +556,22 @@ def add_emv_metadata(post, creditcard)
         post[:metadata][:card_read_method] = creditcard.read_method if creditcard.respond_to?(:read_method)
       end
 
+      def add_shipping_address(post, options = {})
+        return unless shipping = options[:shipping_address]
+
+        post[:shipping] = {}
+
+        post[:shipping][:address] = {}
+        post[:shipping][:name] = shipping[:name] || options[:billing_address][:name]
```

Comment:
Stripe requires the `[:shipping][:name]` field to be present, so I included this fallback to the billing address name in case the request to Spreedly includes a shipping address override without a `name`